service-front/module/Application/config/module.config.php

```diff
@@ -93,16 +93,10 @@
 
     'view_helpers' => [
         'invokables' => [
-            'pageHeaders'           => 'Application\View\Helper\PageHeaders',
-            'elementGroupClass'     => 'Application\View\Helper\ElementGroupClass',
-            'formElementErrors'     => 'Application\View\Helper\FormElementErrors',
```

Comment:
Not sure about some of these, but I thought `formElementErrors` was in use along with the V2 file?

Reply:
I wasn't able to find any uses of this element, but it's possible I have missed one. Are there any places where you know it's used?

Reply:
I can see plenty of places where formElementErrorsV2 is used, but none where formElementErrors is used

Reply:
Can't remember anything specifically for formElementErrors. I've just had a quick look around & can't find anything either.